### PR TITLE
Support expression monitor alerts in mkr alerts list

### PR DESCRIPTION
**Pull request by itchyny**
This pull request adds support for expression monitor alerts. It prints the expression in the configuration, current value, operator and threshold.

### CODE REVIEW

File: alerts.go [hunk 1]
```diff
@@ -163,6 +163,18 @@ func formatJoinedAlert(alertSet *alertSet, colorize bool) string {
 			default:
 				monitorMsg = fmt.Sprintf("%s %.2f > %.2f msec, status:%s", m.Name, alert.Value, m.ResponseTimeCritical, alert.Message)
 			}
+		case *mkr.MonitorExpression:
+			expression := trimExpression(m.Expression)
+			switch alert.Status {
+			case "CRITICAL":
+				monitorMsg = fmt.Sprintf("%s %.2f %s %.2f", expression, alert.Value, m.Operator, m.Critical)
```
Comment (haya14busa) [inline alerts.go]: How about reporting monitor name (`m.Name`) too? I think it's useful especially when the expression is complex.

File: alerts.go [hunk 2]
```diff
@@ -174,11 +186,19 @@ func formatJoinedAlert(alertSet *alertSet, colorize bool) string {
...
+		case "UNKNOWN":
+			statusMsg = "UNKNOWN "
 		}
 	}
 	return fmt.Sprintf("%s %s %s %s%s", alert.ID, time.Unix(alert.OpenedAt, 0).Format(layout), statusMsg, monitorMsg, hostMsg)
 }
 
+var expressionTrimmer = regexp.MustCompile(`[\r\n]+\s*`)
+
+func trimExpression(expr string) string {
```
Comment (haya14busa) [inline alerts.go]: To be precise, this func does not trim expression but make expression one line.
Perhaps, inlineExpression or something might be better as this function name...?

**Comment by itchyny** [comment]
Thanks for your review. I added some tests for mkr alerts list.

File: alerts.go [hunk 3]
```diff
@@ -174,11 +191,20 @@ func formatJoinedAlert(alertSet *alertSet, colorize bool) string {
...
 		case "WARNING":
 			statusMsg = color.YellowString("WARNING ")
+		case "UNKNOWN":
+			statusMsg = "UNKNOWN "
 		}
 	}
 	return fmt.Sprintf("%s %s %s %s%s", alert.ID, time.Unix(alert.OpenedAt, 0).Format(layout), statusMsg, monitorMsg, hostMsg)
 }
 
+var expressionTrimmer = regexp.MustCompile(`[\r\n]+\s*`)
```
Comment (haya14busa) [inline alerts.go]: [nit] It might be better to trim trailing white spaces. `\s*[\r\n]+\s*`

**Comment by itchyny** [comment]
Thank you!

**Pull request merged**